apprentice_teacher.rb

```diff
@@ -1,21 +1,13 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
+require_relative 'educator'
+class ApprenticeTeacher < Educator
+  attr_reader :age, :phase, :target_raise
```

Comment:
Good, took out `:salary` since the superclass has it, but pretty much all of these are also in the superclass -- remove them too.